fastlane/Fastfile

```diff
@@ -69,6 +69,7 @@ ENV["PUBLIC_CONFIG_FILE"]=File.join(PROJECT_ROOT_FOLDER, "config", "Version.Publ
 ENV["INTERNAL_CONFIG_FILE"]=File.join(PROJECT_ROOT_FOLDER, "config", "Version.internal.xcconfig")
 ENV["DOWNLOAD_METADATA"]="./fastlane/download_metadata.swift"
 ENV["PROJECT_ROOT_FOLDER"]=PROJECT_ROOT_FOLDER
+PROJECT_NAME="WordPress-iOS"
```

Comment:
Nit: Maybe name this var `REPOSITORY_NAME` instead, to not confuse it with the Xcode project's name.

Reply:
Alternative idea (take it or leave it): determine this dynamically, using:
```ruby
File.basename(`git remote get-url origin`.chomp, '.git')
```